### PR TITLE
CI: Dump version info in nightly cluster based tests

### PR DESCRIPTION
**Pull request by gavin-stackrox**
## Description

Per title. This information is not well exposed for each cluster flavor.

## Checklist
- [x] Investigated and inspected CI test results

### CODE REVIEW

File: .openshift-ci/pre_tests.py [hunk 1]
```diff
@@ -12,12 +12,23 @@ def run(self):
...
+    VERSIONS_TIMEOUT = 10 * 60
     POLL_TIMEOUT = 60 * 60
 
     def run(self):
+        subprocess.run(
+            [
+                "scripts/ci/lib.sh",
+                "highlight_cluster_versions",
+            ],
+            check=False,
```
Comment (gavin-stackrox) [inline .openshift-ci/pre_tests.py:28]: ignore script failure so this is a best effort info grab

**Review by tommartensen — APPROVED** [review]
LGTM

Where will this artifact show up (artifacts GCS bucket, somewhere on OSCI UI, ...)?

**Pull request merged**